polynote-frontend/style/styles.less

```diff
@@ -296,7 +296,7 @@ body {
 
     .cell-input-editor {
       border: 1px solid @ui-border;
-      border-left-width: 0;
+      border-left: 2pt solid transparent;
```

Comment:
Just curious - any reason to use a border vs a margin or padding? I've never had a good grasp of when to use which. (maybe it doesn't make a difference?)

Reply:
I used border because it already has a margin-left 😛